api/system-openssh/read

```diff
@@ -26,20 +26,48 @@
 
 use JSON;
 use esmith::db;
+use NethServer::ApiTools;
+
 my $db = esmith::db->open("configuration");
 my $port = $db->get_prop('sshd', 'TCPPort');
+my $provider = $db->get_prop('sssd', 'Provider') || 'none';
 my @connections;
 
+sub listGroups {
+  my $groups = `/usr/libexec/nethserver/list-groups -s`;
+  my %groups =  %{ NethServer::ApiTools::safe_decode_json(($groups), {}) };
+  delete ($groups{'domain admins'});
+  return [ keys %groups];
+}
 open(H, "ss -Hn state established '( dport = $port or sport = $port )'|");
 
 while (<H>) {
   ($netid,$recvq,$sendq,$local,$peer) = split;
   push(@connections, {'local' => $local, 'peer' => $peer});
 }
 close(H);
-
 my %out = (
     'status' => { 'connections' => \@connections },
-    'configuration' => $db->prepare_json('sshd')
+    'configuration' => $db->prepare_json('sshd'),
+    'groups' => [@{listGroups()}],
+    'ShellOverrideStatus' => ($db->get_prop('sssd','ShellOverrideStatus') 
+            || 'disabled') eq 'enabled' ? JSON::true : JSON::false,
+    'Provider' => $provider
 );
+
+my %allow = ();
+foreach my $g (split(',',$out{configuration}{props}{AllowGroups})) {
```

Comment:
Please document the `AllowGroups` prop format inside OpenSSH README

Reply:
documented to api